src/components/about-token.js

```diff
@@ -0,0 +1,135 @@
+              />
+            </p>
+            <br />
+            <h3 className="lead">
+              <FormattedMessage
+                id={'about-token.buying-tokens'}
+                defaultMessage={'Where can I buy Origin Tokens?'}
+              />
+            </h3>
+            <h6><strong>
+              <FormattedMessage
+                id={'about-token.exchanges'}
+                defaultMessage={'List of Approved Exchanges'}
+              /></strong>
+            </h6>
+            <p>
+              <FormattedMessage
+                id={'about-token.exchanges-text'}
+                defaultMessage={'The following are the only approved exchanges where you can buy Origin Tokens. Never attempt to buy anywhere else.'}
+              />
+            </p>
+            <p><a href="#">
+              <FormattedMessage
+                id={'about-token.exchange-1'}
+                defaultMessage={'Exchange Name >'}
+              /></a>
+            </p>
+            <p><a href="#">
+              <FormattedMessage
+                id={'about-token.exchange-2'}
+                defaultMessage={'Exchange Name >'}
+              /></a>
+            </p>
+            <p><a href="#">
+              <FormattedMessage
+                id={'about-token.exchange-3'}
+                defaultMessage={'Exchange Name >'}
+              /></a>
+            </p>
+            <br />
+            <h3 className="lead">
+              <FormattedMessage
+                id={'about-token.earn'}
+                defaultMessage={'Can I earn Origin Tokens?'}
+              />
+            </h3>
+            <h6><strong>
+              <FormattedMessage
+                id={'about-token.earning'}
+                defaultMessage={'Earning Origin Tokens'}
+              /></strong>
+            </h6>
+            <p>
+              <FormattedMessage
+                id={'about-token.earning-text'}
+                defaultMessage={'You can earn Origin Tokens by following any of the steps below:'}
+              />
+            </p>
+            <ul>
+              <li>
+                <FormattedMessage
+                  id={'about-token.earn-step-1'}
+                  defaultMessage={'Do this to earn tokens'}
+                />
+              </li>
+              <li>
+                <FormattedMessage
+                  id={'about-token.earn-step-2'}
+                  defaultMessage={'Do this to earn tokens'}
+                />
+              </li>
+              <li>
+                <FormattedMessage
+                  id={'about-token.earn-step-3'}
+                  defaultMessage={'Do this to earn tokens'}
+                />
+              </li>
+            </ul>
+          </div>
+          <div className="col-md-6">
```

Comment:
@mek32390 this should have a class with some `padding-top` instead of using line breaks.